presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveStorageFormats.java

```diff
@@ -228,12 +231,26 @@ private static void setSessionProperties(Map<String, String> sessionProperties)
     {
         Connection connection = defaultQueryExecutor().getConnection();
         try {
-            PrestoConnection prestoConnection = connection.unwrap(PrestoConnection.class);
-            // create more than one split
-            prestoConnection.setSessionProperty("task_writer_count", "4");
-            prestoConnection.setSessionProperty("redistribute_writes", "false");
-            for (Map.Entry<String, String> sessionProperty : sessionProperties.entrySet()) {
-                prestoConnection.setSessionProperty(sessionProperty.getKey(), sessionProperty.getValue());
+            if (usingFacebookJdbcDriver()) {
```

Comment:
Please extract

```java
JDBCDriverUtils.setSessionProperty(Connection connectino, String key, String value)
```

which does conditional logic, and use it here.

Also shouldn't `usingFacebookJdbcDriver()` and `usingSimbaJdbcDriver()` take connection as argument? It does not seem right to me that those are parameterless static methods.